libs/features/mas/src/variants/variants.js

```diff
@@ -52,6 +53,12 @@ registerVariant(
     SPECIAL_OFFERS_AEM_FRAGMENT_MAPPING,
     SpecialOffer.variantStyle,
 );
+registerVariant(
```

Comment:
is this variant needed for adobe.com or MAS consumers? if so, please register it in mas.js.
The goal is to keep the size of merch-card.js for Milo under control.

Reply:
The answer is no. I have moved the registry to `merch-card.js`.